### PR TITLE
delete from storage

### PR DESCRIPTION
**Pull request by komalg1**
## Purpose

* ...

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: code/backend/pages/03_Delete_Data.py [hunk 1]
```diff
@@ -65,6 +66,9 @@
                 files_to_delete = search_handler.delete_files(
                     selected_files,
                 )
+                # Delete from Storage
+                blob_client = AzureBlobStorageClient()
+                blob_client.delete_files(selected_files)
```
[the remark this answers is not in the document]
Comment (komalg1) [reply code/backend/pages/03_Delete_Data.py:71]: Yes tested this. It picks up the filename correctly with "/" for urls

[the remark this answers is not in the document]
Comment (adamdougal) [reply code/backend/pages/03_Delete_Data.py:71]: nice

Comment (superhindupur) [inline code/backend/pages/03_Delete_Data.py:71]: Now that @liammoat is handling the blob delete event to delete the document from the search index under https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/pull/893, do you think we should just do the blob delete from here, and let the function trigger handle the index delete?

**Comment by superhindupur** [comment]
Would help to have the purpose etc on the PR description, and a more descriptive title for the PR please. Thanks!

File: code/backend/pages/03_Delete_Data.py (hunk 1 above)
Comment (komalg1) [reply code/backend/pages/03_Delete_Data.py:71]: Yes will close this PR for now and wait for @liammoat changes to go in

**Pull request closed without merging**